index.js

```diff
@@ -55,6 +55,56 @@ server.get("/login", (request, response) => {
   response.sendFile(__dirname+"/views/login.html");
 });
 
+server.get("/cadastroUser", (request, response) => {
+  var emailMsn = "Insira um email:";
+  var password1Msn = "Insira sua senha:";
+  var password2Msn = "Confirme a senha:";
+
+  const html = jetpack.read(__dirname+"/views/cadastroUser.html");
+  const formattedHtml = Eta.render(html, { emailMsn, password1Msn, password2Msn });
+
+  response.send(formattedHtml);
+});
+
+server.post("/cadastroUser", (request, response) => {
```

Comment:
Nome de URL não deve possuir letra maiúscula, prefira usar assim:

```suggestion
server.post("/cadastro-user", (request, response) => {
```